db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peep(id SERIAL PRIMARY KEY, peep VARCHAR(500), timestamp TIMESTAMPTZS DEFAULT NOW())
```

Comment:
I found a couple of typos in here
```suggestion
CREATE TABLE peep(id SERIAL PRIMARY KEY, peep VARCHAR(500), timestamp TIMESTAMPTZ DEFAULT NOW());
```